src/classes/Countertop.js

```diff
@@ -14,6 +15,8 @@ class Countertop {
 
 	stations = []
 
+	state = countertopStates.STOPPED
```

Comment:
I forget if we decided it's bad / error prone to set values here vs in the constructor